### PR TITLE
Enhancement/Modify word-wrap for toasts to show whole messages & String replace psycopg2

### PR DESCRIPTION
**Pull request by exzizt**
* Modifies the style of our notification messages to better fit long text
* Replaces occurrences of "psycopg2" with "Database Error" for our "custom axios" error messages
image

### CODE REVIEW

**Comment by pdittaro** [comment]
It might be nice to move towards a header/content in our toast messages? E.g. large 'Database Error', then smaller, potentially scrollable/code block error message?

File: services/core-web/common/customAxios.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ promiseFinally.shim();
 const UNAUTHORIZED = 401;
 const MAINTENANCE = 503;
 
+const formatErrorMessage = (errorMessage) => {
+  errorMessage = errorMessage.replace("(psycopg2.", "(DatabaseError.");
```
Comment (nayr974) [inline services/core-web/common/customAxios.js:14]: is it worth catching "psycopg2" vs "psychopg2." ? It might exist in other places in a string?

**Review by nayr974 — APPROVED** [review]

File: services/core-web/common/customAxios.js (hunk 1 above)
Comment (exzizt) [reply services/core-web/common/customAxios.js:14]: We can do that, I thought we may just want to change the initial mention of it though. Happy to have to replace all instances if that's what we want?

**Comment by exzizt** [comment]
> It might be nice to move towards a header/content in our toast messages? E.g. large 'Database Error', then smaller, potentially scrollable/code block error message?

A good idea, but this would require a lot more time and work to figure out, of course. A good UX-debt item.

File: services/core-web/common/customAxios.js (hunk 1 above)
Comment (nayr974) [reply services/core-web/common/customAxios.js:14]: initial mention is probably preferable, so long as it's consistently followed by the period

Comment (exzizt) [reply services/core-web/common/customAxios.js:14]: Yup, and by the initial "(".

**Pull request merged**